### PR TITLE
remove VSA and Ceph references

### PR DESCRIPTION
**Pull request by csymons-suse**
in Operations Guide
bsc#1076227 Priority 1

### CODE REVIEW

**Comment by csymons-suse** [comment]
this should be good now. Conflicting files have been removed.

**Comment by dmpop** [comment]
Thanks, but you still need to rebase.

File: xml/operations-blockstorage-creating_voltype.xml [hunk 1]
```diff
@@ -76,7 +74,7 @@
    <step>
     <para>
      Enter a unique name for the volume type and then click the
-     <emphasis role="bold">Create Volume Type</emphasis> button to complete the
+     <guimenu role="bold">Create Volume Type</guimenu> button to complete the
```
Comment (dmpop) [inline xml/operations-blockstorage-creating_voltype.xml]: Replace \<emphasis role="bold"> with \<guimenu>

File: xml/operations-blockstorage-creating_voltype.xml [hunk 2]
```diff
@@ -92,51 +90,40 @@
...
      Log into the Horizon dashboard. See <xref linkend="cloudadmin_gui"/> for
      details on how to do this.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Ensure that you are scoped to your <emphasis role="bold">admin</emphasis>
-     Project. Then under the <emphasis role="bold">Admin</emphasis> menu in the
```
Comment (dmpop) [inline xml/operations-blockstorage-creating_voltype.xml]: Replace *<emphasis>* with *<guimenu>* for all menu items.

File: xml/operations-blockstorage-creating_voltype.xml [hunk 3]
```diff
@@ -182,11 +168,11 @@
       </imageobject>
      </mediaobject>
     </informalfigure>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Click the <emphasis role="bold">Create</emphasis> button on the
-     <emphasis role="bold">Volume Type Extra Specs</emphasis> screen.
```
Comment (dmpop) [inline xml/operations-blockstorage-creating_voltype.xml]: Replace \<emphasis role="bold"> with \<guimenu>

File: xml/operations-blockstorage-creating_voltype.xml [hunk 4]
```diff
@@ -92,51 +90,40 @@
...
      details on how to do this.
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     Ensure that you are scoped to your <emphasis role="bold">admin</emphasis>
-     Project. Then under the <emphasis role="bold">Admin</emphasis> menu in the
-     navigation pane, click on <emphasis role="bold">Volumes</emphasis> under
```
Comment (dmpop) [inline xml/operations-blockstorage-creating_voltype.xml]: Replace \<emphasis role="bold"> with \<guimenu>

File: xml/operations-blockstorage-creating_voltype.xml [hunk 5]
```diff
@@ -164,13 +150,13 @@
...
     </informalfigure>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     In the <emphasis role="bold">Actions</emphasis> column of the Volume Type
-     you created earlier, click the drop-down option and select
-     <emphasis role="bold">View Extra Specs</emphasis> which will bring up the
-     <emphasis role="bold">Volume Type Extra Specs</emphasis> options.
```
Comment (dmpop) [inline xml/operations-blockstorage-creating_voltype.xml]: If this an GUI element, replace \<emphasis role="bold"> with \<guimenu>

**Comment by sknorr** [comment]
Thanks!

**Pull request merged**